eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -203,6 +208,23 @@ delayingAction TreasuryWithdrawals {} = False
 delayingAction ParameterChange {} = False
 delayingAction InfoAction {} = False
 
+actionPriority :: GovAction era -> Int
```

Comment:
Why not implement `Ord`? are we using this value somewhere other than for reordering? Or maybe if we had an instance of `Ord`  there would be other - undesired - consequences?

Reply:
Because it is priority, not `Ord`. 
`Ord` instance requires to use all of the contents of the type, otherwise you'll break referential transparency

Reply:
Ah yes. makes sense ,thanks